testrunner-core-plugin/src/main/scala/edu/illinois/cs/testrunner/coreplugin/TestPluginUtil.java

```diff
@@ -0,0 +1,94 @@
+package edu.illinois.cs.testrunner.coreplugin;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Paths;
+import java.net.URL;
+import java.net.URLClassLoader;
+import java.util.logging.Logger;
+import java.util.Arrays;
+import java.util.List;
+import java.util.logging.Level;
+
+import edu.illinois.cs.testrunner.util.ProjectWrapper;
+import edu.illinois.cs.testrunner.configuration.ConfigProps;
+import edu.illinois.cs.testrunner.configuration.Configuration;
+
+public class TestPluginUtil {
+  final public static String pluginName = "testplugin";
```

Comment:
nit: 4 spaces indentation to keep the style consistent?

Reply:
Oops, it's my bad. Thanks for figuring this out!